src/Features/VisualBasic/Portable/InitializeParameter/VisualBasicInitializeMemberFromParameterCodeRefactoringProvider.vb

```diff
@@ -29,8 +29,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.InitializeParameter
             Return InitializeParameterHelpers.TryGetLastStatement(blockStatement)
         End Function
 
-        Protected Overrides Function GetBlockOperation(functionDeclaration As SyntaxNode, semanticModel As SemanticModel, cancellationToken As CancellationToken) As IBlockOperation
-            Return InitializeParameterHelpers.GetBlockOperation(functionDeclaration, semanticModel, cancellationToken)
+        Protected Overrides Function TryGetBlockOperation(functionDeclaration As SyntaxNode, semanticModel As SemanticModel, cancellationToken As CancellationToken, ByRef blockOperation As IBlockOperation) As Boolean
+            blockOperation = InitializeParameterHelpers.GetBlockOperation(functionDeclaration, semanticModel, cancellationToken)
+            Return blockOperation IsNot Nothing
```

Comment:
i don't like that C# and VB are starting to diverge between impls.

Reply:
Yeah, this one was tricky because C# still offers the refactoring even if the block is null.  So in order to distinguish between "null, but go ahead and refactor" and "null, don't refactor" I added the boolean.  VB didn't need this distinction, but I added the boolean to keep them in sync.  

Since I wasn't able to repro this in C# (thus no test was added) perhaps I should just do a simpler fix for VB and leave C# alone.

Reply:
Yes please :)